server/notebooks/api_views.py

```diff
@@ -62,6 +62,11 @@ def get_serializer_class(self):
             return NotebookRevisionSerializer
         return NotebookRevisionDetailSerializer
 
+    def perform_destroy(self, instance):
+        if instance.notebook.owner != self.request.user:
+            raise PermissionDenied
+        viewsets.ModelViewSet.perform_destroy(self, instance)
```

Comment:
Is this method override actually necessary? I believe this should be taken care of by the permission classes, above:

https://github.com/iodide-project/iodide/pull/904/files#diff-6c8e78b546a064d4a9debd0afe22881dR45

Reply:
Oh wait, on second look, no it isn't. Definitely need this logic. ;)